pagespeed/system/system_tests/ipro_noop.sh

```diff
@@ -48,9 +48,20 @@ URL1="${URL}&PageSpeedNoop=$RANDOM1"
 URL2="${URL}&PageSpeedNoop=$RANDOM2"
 fetch_until "$URL1" "grep -c image/webp" 1 --save-headers
 #NUM_REWRITES_URL1=$(scrape_stat image_rewrites)
+echo -n Waiting for quiescence by checking serf_fetch_active_count ...
+# The last check in this test was observed to flake.Let's see if waiting
+# for quiescence here stabilizes it.
+while [ $(scrape_stat serf_fetch_active_count) -gt 0 ]; do
```

Comment:
It would be good to keep a count of active rewrite drivers as well for this purpose.  Serf is only one of the things that can finish asynchronously; slow rewrites can as well.

This can be a TODO as this will probably help and I don't want to block you.  And keeping an active rewrite_count is not necessarily trivial.